src/writer/vote.ts

```diff
@@ -132,6 +147,21 @@ export async function action(body, ipfs, receipt, id): Promise<void> {
   ]);
 
   // Store vote in dedicated table
+  const params = {
+    id,
+    ipfs,
+    voter,
+    created: parseInt(msg.timestamp),
+    space: msg.space,
+    proposal: msg.payload.proposal,
+    choice: JSON.stringify(msg.payload.choice),
+    metadata: JSON.stringify(msg.payload.metadata || {}),
+    vp,
+    vp_by_strategy: JSON.stringify([]),
+    vp_state: '',
```

Comment:
These can't also be set right away?

Reply:
Yes i'm doing it on another branch but its still a draft